cmake/modules/MacOSXBundleInfo.plist.in

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleIconFile</key>
 	<string>flameshot</string>
 	<key>CFBundleIdentifier</key>
-	<string>https://flameshot.org/</string>
+	<string>https://github.com/namecheap/flameshot/releases/latest</string>
```

Comment:
Should we change this back for the flameshot-org project?

Reply:
Sure, it shouldn't be Namecheap here, I missed it, sorry.